src/ert/scheduler/lsf_driver.py

```diff
@@ -149,9 +149,12 @@ async def kill(self, iens: int) -> None:
         job_id = self._iens2jobid[iens]
 
         logger.debug(f"Killing realization {iens} with LSF-id {job_id}")
+
+        bkill_cmd_with_signals = f"{self._bkill_cmd} -s SIGTERM {job_id}; sleep 30; {self._bkill_cmd} -s SIGKILL {job_id};"
```

Comment:
If we have tests asserting the output of this bkill (I have not checked), these tests will take 30 seconds I suppose?

Reply:
I will put the sleep period as an instance variable. Or do you think it would be better as a class variable? I see we already use both in lsf_driver.